recipes-domx/meta-xt-prod-vgpu/recipes-graphics/images/core-image-weston.bbappend

```diff
@@ -0,0 +1,3 @@
+IMAGE_INSTALL_append = " \
+    kernel-modules \
+"
```

Comment:
Is the "recipes-domx/meta-xt-prod-vgpu/recipes-graphics/images/core-image-weston.bbappend" right place to ask build system to install "kernel" modules?

Reply:
Yes, since we want modules in both domains. From here we make influence for both domains from one place.

Reply:
It is quite strange, that yocto does not install kernel modules by default.